packages/ibm-products/src/components/Coachmark/Coachmark.stories.jsx

```diff
@@ -79,6 +80,25 @@ export default {
       description: 'Tooltip text and aria label for the Close button icon.',
     },
   },
+  decorators: [
+    (story) => (
+      <Annotation
```

Comment:
super minor, but the story is slightly cut off in the [overview preview](https://deploy-preview-8634--carbon-for-ibm-products.netlify.app/?path=/docs/deprecated-coachmark-coachmark--overview#floating-example-usage). can you take a quick look at it?

<img width="1058" height="534" alt="Screenshot 2025-11-14 at 10 43 30 AM" src="https://github.com/user-attachments/assets/2f111b43-8c4e-4848-8a01-c6c5e40bea16" />